google/cloud/forseti/common/util/logger.py

```diff
@@ -44,12 +50,16 @@ def get_logger(module_name):
     Returns:
         logger: An instance of the configured logger.
     """
-    # TODO: Move this into a configuration file.
-    syslog_handler = logging.handlers.SysLogHandler()
-    syslog_handler.setFormatter(logging.Formatter(SYSLOG_LOG_FMT))
+
+    if os.path.exists('/var/log/forseti.log'):  # ubuntu on GCE
+        default_log_handler = logging.FileHandler('/var/log/forseti.log')
+    else:
+        default_log_handler = logging.handlers.SysLogHandler()
+
+    default_log_handler.setFormatter(logging.Formatter(SYSLOG_LOG_FMT))
```

Comment:
We probably want to include a the timestamp as '%(asctime)s' before the levelname for the messages in forseti.log, so you need two formats, one for syslog, and one for forseti.log.

Reply:
![image](https://user-images.githubusercontent.com/6977544/39940846-700dc136-550f-11e8-81d7-d33e8e80b67c.png)

Reply:
Just a nit, this will cause the timestamp to be output twise for the SysLogHandler since syslog also adds a timestamp. I'm not sure that's really a problem.

Reply:
Done, we came this far, might as well take care of this.  :)